src/language-css/print/comma-separated-value-group.js

```diff
@@ -147,11 +147,17 @@ function printCommaSeparatedValueGroup(path, options, print) {
       continue;
     }
 
-    // styled.div` background: var(--${one}); `
+    // We should keep spaces between words in a embedded JS expression
```

Comment:
We can't know from the AST whether we can remove the spaces or whether we have to preserve them, so I believe we need to use the whitespace information from the original text.